src/CattyTests/Bricks/WaitBrickTests.swift

```diff
@@ -73,7 +73,7 @@ final class WaitBrickTests: XCTestCase {
                                                execOnMainRunLoop: true,
                                                startTimerImmediately: true) { _ in }
         self.scheduler.registerTimer(extendedTimer)
-        XCTAssertEqual(scheduler._activeTimers.count, 1)
+        XCTAssertEqual(scheduler.synchronizedTimerArray.count, 1)
     }
 
     func testSchedulerRemoveTimer_activeTimersCountIsZero() {
```

Comment:
Please remove the deprecated `_activeTimers` .

Reply:
Also removed this one.